client/i18n/locales/english/trending.json

```diff
@@ -1,62 +1,63 @@
+  "article2title":"Python For Loop",
+  "article2link":"https://www.freecodecamp.org/news/python-for-loop-for-i-in-range-example/",
+  "article3title":"What is JavaScript?",
+  "article3link":"https://www.freecodecamp.org/news/what-is-javascript-definition-of-js/",
+  "article4title":"Learn How To Code",
+  "article4link":"https://www.freecodecamp.org/news/how-to/",
+  "article5title":"Chrome Bookmarks",
+  "article5link":"https://www.freecodecamp.org/news/chrome-bookmarks-location-guide-how-to-delete-or-recover-a-bookmark/",
+  "article6title":"Concatenate Excel",
+  "article6link":"https://www.freecodecamp.org/news/concatenate-in-excel-how-to-combine-text-and-concat-strings/",
+  "article7title":"C# String to Int",
+  "article7link":"https://www.freecodecamp.org/news/how-to-convert-a-string-to-an-int-in-c-tutorial-with-example-code/",
+  "article8title":"Git Switch Branch",
+  "article8link":"https://www.freecodecamp.org/news/git-switch-branch/",
+  "article9title":"JavaScript Splice",
+  "article9link":"https://www.freecodecamp.org/news/javascript-splice-how-to-use-the-splice-js-array-method/",
+  "article10title":"HTML Link",
+  "article10link":"https://www.freecodecamp.org/news/html-button-link-code-examples-how-to-make-html-hyperlinks-using-the-href-attribute-on-tags/",
+  "article11title":"Bayes Rule",
+  "article11link":"https://www.freecodecamp.org/news/bayes-rule-explained/",
+  "article12title":"Python Map",
+  "article12link":"https://www.freecodecamp.org/news/python-map-function-how-to-map-a-list-in-python-3-0-with-example-code/",
+  "article13title":"HTML Italics",
+  "article13link":"https://www.freecodecamp.org/news/html-italics-tutorial-how-to-make-text-italic-with-the-i-tag/",
+  "article14title":"Python SQL",
+  "article14link":"https://www.freecodecamp.org/news/python-sql-how-to-use-sql-databases-with-python/",
+  "article15title":"HTML Bold",
+  "article15link":"https://www.freecodecamp.org/news/html-bold-text-tutorial-how-to-use-the-b-tag/",
+  "article16title":"GraphQL VS Rest",
+  "article16link":"https://www.freecodecamp.org/news/graphql-vs-rest-benefits-and-code-example-comparisons/",
+  "article17title":"If Function Excel",
+  "article17link":"https://www.freecodecamp.org/news/if-function-excel-tutorial-and-how-to-do-multiple-if-statements-in-excel/",
+  "article18title":"HTML List",
+  "article18link":"https://www.freecodecamp.org/news/html-list-how-to-use-bullet-points-ordered-and-unordered-lists/",
+  "article19title":"Wav File",
+  "article19link":"https://www.freecodecamp.org/news/wav-file-format-how-to-open-a-wav-and-convert-wavs-to-mp3/",
+  "article20title":"Model View Controller ",
+  "article20link":"https://www.freecodecamp.org/news/the-model-view-controller-pattern-mvc-architecture-and-frameworks-explained/",
+  "article21title":"Git Checkout Remote Branch",
+  "article21link":"https://www.freecodecamp.org/news/git-checkout-remote-branch-tutorial/",
+  "article22title":"Insert checkbox in word",
```

Comment:
```suggestion
  "article22title":"Insert Checkbox in Word",
```

Reply:
it is a convention we have with the editorial team.